src/exporter/trace/stdout/mod.rs

```diff
@@ -0,0 +1,49 @@
+use crate::exporter::trace;
+use std::any;
+use std::sync::Arc;
+
+#[derive(Debug)]
+/// Exporter
+pub struct Exporter {}
+
+#[derive(Debug)]
```

Comment:
```suggestion
#[derive(Debug, Default)]
pub struct Exporter {}

impl Builder {
  pub fn new() -> Self {
    Self {}
  }
  
  pub fn build() -> Exporter {
    Exporter {}
  }
}
```

Reply:
I'm replacing build to init(), so we can have the same methods as the jaeger export in the Builder